File: Exp288/Exp288/Form1.vb
Imports System.Data.SqlClient
Imports System.Windows.Forms.VisualStyles.VisualStyleElement

Public Class Form30
    Dim con As SqlConnection = New SqlConnection("Data Source=MAYUR\SQLEXPRESS;Initial Catalog=exp28;Integrated Security=True")
    Dim q As SqlCommand
Dim a As New DataSet
Dim ad As SqlDataAdapter
    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        con.Open()
        q = New SqlCommand("select * from e28", con)
        ad = New SqlDataAdapter(q)
        ad.Fill(a, "e28")
        TextBox1.DataBindings.Add("text", a, "e28.sname")
        TextBox2.DataBindings.Add("text", a, "e28.clgname")
    End Sub
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
Me.BindingContext(a, "e28").Position = 0
End Sub
Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
Me.BindingContext(a, "e28").Position += 1
End Sub
Private Sub Button3_Click(sender As Object, e As EventArgs) Handles Button3.Click
Me.BindingContext(a, "e28").Position = Me.BindingContext(a, "e28").Count - 1
End Sub
Private Sub Button4_Click(sender As Object, e As EventArgs) Handles Button4.Click
Me.BindingContext(a, "e28").Position -= 1
End Sub
End Class

File: Accounts/Accounts/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Configuration;
using System.Data;
using System.Data.OleDb;
using System.Data.SqlClient;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Accounts
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {

            /* OleDbConnection conn = new OleDbConnection("Provider=Microsoft.ACE.OLEDB.12.0;Data Source=D:\\AP2\\AP2.mdb");
             if (conn.State == ConnectionState.Closed)
             {
                 conn.Open();
             }
             OleDbCommand cmd = new OleDbCommand("SELECT * FROM [ap2-0021]", conn);
             OleDbDataAdapter da = new OleDbDataAdapter();
             da.SelectCommand = cmd;
             DataTable dt = new DataTable();
             dt.Clear();
             da.Fill(dt);
             dataGridView1.DataSource = dt;
             conn.Close();
            */
            string connectionString = ConfigurationManager.ConnectionStrings["SQLConnectionString"].ToString();// "Data Source=MAYUR\\SQLEXPRESS; Initial Catalog=student;integrated Security=SSPI;";


            using (SqlConnection _con = new SqlConnection(connectionString))
            {
                string queryStatement = "SELECT TOP 5 * FROM dbo.stu";

                using (SqlCommand _cmd = new SqlCommand(queryStatement, _con))
                {
                    DataTable customerTable = new DataTable("Top5Customers");
                    SqlDataAdapter _dap = new SqlDataAdapter(_cmd);

                    _con.Open();
                    _dap.Fill(customerTable);
                    _con.Close();

                }
            }
        }
    }
}
           
